vertx-mongo-client/pom.xml

```diff
@@ -79,4 +79,13 @@
     </plugins>
   </build>
 
+  <dependencies>
```

Comment:
Why not use the JUnit 4 parameterised runner?

Reply:
It looked more verbose. Also, I'm not sure if it's possible to parametrize over the method, not entire class. I can try to refactor that.

Reply:
It's no biggy really - @johnoliver noted that this is slightly cleaner than the JUnit 4 way